modules/core/src/lib/layer.js

```diff
@@ -122,10 +122,11 @@ export default class Layer extends Component {
   }
 
   // This layer needs a deep update
-  // TODO - Need to align with existing needsUpdate before uncommenting
-  // For now async props will call layerManager directly
-  setLayerNeedsUpdate() {
-    this.context.layerManager.setNeedsUpdate(String(this));
+  setNeedsUpdate(update = true) {
```

Comment:
Nit: Originally I didn't want to give applications the ability to turn the `needsUpdate` flag off, so I intentionally did not provide a parameter to this call. 

It should only be reset by the framework in a single place and we can set the flag directly there, no need to call a public method.